README.md

```diff
@@ -73,6 +73,94 @@ Add connection to `config/queue.php`:
 ],
 ```
 
+### Optional Config
+
+Optionally add queue options to the config of a connection. 
+Every queue created for this connection, get's the properties.
+
+When you want to prioritize messages when they were delayed, then this is possible by adding extra options.
+- When max-priority is omitted, the max priority is set with 2 when used.
+
+```php
+'connections' => [
+    // ...
+
+    'rabbitmq' => [
+        // ...
+
+        'options' => [
+            'queue' => [
+                // ...
+
+                'prioritize_delayed_messages' =>  false,
+                'queue_max_priority' => 10,
```

Comment:
Is it possible to every job has it's own priority in a queue?

Reply:
Yes this is possible.

When we delay messages in RabbitMQ by sending them to a queue with DLX, the message (when expires) is put back at the **end** of the queue.

Case: 
When the queue holds more than a 1000 messages (Jobs), the delayed job is processed again after the sum of (delayTime + (processingTime * 1000)).

To make sure the delayed job is processed as soon as it comes out of the delayQueue, a higher priority is set and rabbitMQ makes sure the delayed message is transferred to the **front** of the queue.

When working with one delayTime (one DelayQueue), 2 priority levels are sufficient. When implementing a delayStrategy (multiple delayQueues), you probably want more priority levels.
This to make sure the oldest Job is processed first or as soon as possible.

Reply:
May we we can add it in every job. Like through properties or even method.

Reply:
Prioritizing Jobs from within a Job can be done by dispatching the job to an other queue. (hig priority queues. (Laravel mentions this in the documentation.)

When a job fails, u can dispatch the Job to a queue which will be processed first by the workers. there is a method `failed()` u can use for this.

Maybe i miss understood the question: But i don't think such a feature should be part of this library.